### PR TITLE
feat: Segmentation mode adding subAttribute / feat: Segmentation mode supports rect selection

### PR DESCRIPTION
**Pull request by Asliuyong**

### CODE REVIEW

File: packages/lb-annotation/src/core/pointCloud/store/index.ts [hunk 1]
```diff
@@ -719,6 +722,20 @@ class PointCloudStore {
...
     this.currentAttribute = attribute;
   }
+
+  public setSubAttribute(key: string, value: string) {
+    if (!this.cacheSegData) {
+      return;
+    }
+    this.cacheSegData.subAttribute = {
+      ...this.cacheSegData.subAttribute,
+      [key]: value,
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/pointCloud/store/index.ts:732]: Need to distinguish between incremental additions and full-scale changes.

Comment (Asliuyong) [reply packages/lb-annotation/src/core/pointCloud/store/index.ts:732]: Can only be added or replaced without deletion, so there is no need to distinguish

File: packages/lb-annotation/src/core/pointCloud/store/index.ts [hunk 2]
```diff
@@ -719,6 +722,20 @@ class PointCloudStore {
   public setAttribute(attribute: string) {
     this.currentAttribute = attribute;
   }
+
+  public setSubAttribute(key: string, value: string) {
+    if (!this.cacheSegData) {
+      return;
+    }
+    this.cacheSegData.subAttribute = {
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/pointCloud/store/index.ts:730]: This line needs to be compatible with the original data where `subAttribute` may not exist

**Pull request merged**